### PR TITLE
Remove disable flag

### PR DESCRIPTION
**Pull request by laurieontech**
Thought through this a lot, but I don't think we need both this and ignorePreset since this is unlikely to be used going forward without a combination of the switch toggle and a built-in preset.

Also included an updated README since explaining this behavior to Max was sufficiently confusing!

### CODE REVIEW

File: packages/gatsby-theme-notes/README.md [hunk 1]
```diff
@@ -59,3 +59,29 @@ module.exports = {
 | `mdxOtherwiseConfigured` | `false`          | Set this flag `true` if `gatsby-plugin-mdx` is already configured for your site. |
 | `homeText`               | `~`              | Root text for notes breadcrumb trail                                             |
 | `breadcrumbSeparator`    | `/`              | Separator for the breadcrumb trail                                               |
+| `preset`  | `gatsby-theme-ui-preset`          | Install your dependency and pass in the package name. All local shadowing will deep merge with this preset. |
+| `ignorePreset`  | `false`          | Set this flag `true` if you want to use the blog theme without the default `gatsby-theme-ui-preset` styles. |
```
Comment (jxnblk) [inline packages/gatsby-theme-notes/README.md:63]: I might be forgetting a detail with how this is implemented, but could we use the `preset` option for this as well (i.e. combine them into the same option), e.g. `preset: false`?

Comment (laurieontech) [reply packages/gatsby-theme-notes/README.md:63]: Technically this is the same as passing an empty object to preset. In my mind the explicit flag option seemed clearer, but checking for a boolean value in preset is really nice DX. I'm happy to do that if we don't think it'll be confusing.

Comment (jxnblk) [reply packages/gatsby-theme-notes/README.md:63]: I don't have any examples off hand, but pretty sure this is a pattern I've seen in other configs/CLIs -- so I'm biased, but it doesn't seem too confusing...

Comment (laurieontech) [reply packages/gatsby-theme-notes/README.md:63]: Updated the README for this PR. The actual implementation of ignorePreset is in a separate PR which was a mistake/confusing. So I'll make that functionality change there.

**Pull request merged**